dev/integration_tests/ios_platform_view_tests/test_driver/main_test.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart' hide TypeMatcher, isInstanceOf;
+
+Future<void> main() async {
+  FlutterDriver driver;
+
+  setUpAll(() async {
+    driver = await FlutterDriver.connect();
+  });
+
+  tearDownAll(() => driver.close());
+
+  test('Merge thread to create and remove platform views should not crash', () async {
+
+    final SerializableFinder platformViewButton =
+        find.byValueKey('platform_view_button');
+    await driver.waitFor(platformViewButton);
+    await driver.tap(platformViewButton);
+
+    final SerializableFinder plusButton =
+        find.byValueKey('plus_button');
+    await driver.waitFor(plusButton);
+    await driver.waitUntilNoTransientCallbacks();
+    await driver.tap(plusButton);
+    await driver.waitUntilNoTransientCallbacks();
+
+    final SerializableFinder backButton = find.pageBack();
+    await driver.tap(backButton);
+    await driver.waitUntilNoTransientCallbacks();
+
+    final Health driverHealth = await driver.checkHealth();
+    expect(driverHealth.status, HealthStatus.ok);
+  });
+
+    test('Merge thread to create and remove platform views should not crash', () async {
+
+    final SerializableFinder platformViewButton =
+        find.byValueKey('platform_view_button');
+    await driver.waitFor(platformViewButton);
+    await driver.tap(platformViewButton);
+
+    await driver.waitUntilNoTransientCallbacks();
+
+    final SerializableFinder backButton = find.pageBack();
+    await driver.tap(backButton);
+    await driver.waitUntilNoTransientCallbacks();
+
+    final SerializableFinder no_action_button =
+        find.byValueKey('no_action_button');
+    await driver.waitFor(no_action_button);
+    await driver.tap(no_action_button);
```

Comment:
Ohh now I see, still worth to put a comment on that button explaining this is why it's there.

Reply:
one